python/device/device.py

```diff
@@ -61,9 +61,9 @@
 # set default logging which will only go to the console
 
 logging.basicConfig(level=logging.WARNING)
-logging.getLogger("paho").setLevel(level=logging.DEBUG)
+# logging.getLogger("paho").setLevel(level=logging.DEBUG)
```

Comment:
Is this meant to be commented out? Same with line 66

Reply:
Yes, I meant to leave these commented out